src/content/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model.mdx

```diff
@@ -46,15 +42,13 @@ For users on our [original user model](/docs/accounts/original-accounts-billing/
 
 ## Requirements [#who-is-doc-for]
 
-This doc and the surrounding [section of docs](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles) shows you how to manage users who are on our [original user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models). If you were a New Relic customer before July 30 2020, you likely have users on our original user model (and not the [New Relic One user model](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles)).
+This doc and the surrounding [section of docs](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles) shows you how to manage users on our [original account/user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models). If you were a New Relic customer before July 30 2020 and haven't [migrated your users to the new model](/docs/accounts/original-accounts-billing/original-users-roles/user-migration), your users are on our original account/user model (and not the [New Relic One model](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles)).
 
-One way to quickly check your users' user model: if you can see users in the [**Users and roles** UI](#view-access), those users are on our original user model.
-
-Want to learn more about user model changes? See [Overview of user models](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-users/transition-guide-new-pricing-plan).
+If you're an admin and want to check to see if you have users on the original model: if you can see users in the [**Users and roles** UI](#view-access), those users are on our original user model. 
```

Comment:
"if you want to check to see" reads a little redundant, I'd remove one of the verbs

Reply:
Also, If, capital I, after the :